03_build_installer.sh

```diff
@@ -20,7 +20,11 @@ function build_installer() {
   # Build installer
   pushd .
   cd $OPENSHIFT_INSTALL_PATH
-  TAGS="${OPENSHIFT_INSTALLER_BUILD_TAGS:-libvirt baremetal}" DEFAULT_ARCH=$(get_arch) hack/build.sh
+  local default_tags="libvirt baremetal"
+  if [ "${FIPS_MODE:-false}" = "true" && "${FIPS_VALIDATE:-false}" = "true" ]; then
```

Comment:
You're mixing shell syntaxes here, I believe. Either

 ` [ "${FIPS_MODE:-false}" = "true" ] && [ "${FIPS_VALIDATE:-false}" = "true" ]`

or

 ` [[ "${FIPS_MODE:-false}" == "true" && "${FIPS_VALIDATE:-false}" == "true" ]]`

Reply:
You're right, I misread the bit in the man page where it says not to use `-a`